Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -70,6 +70,11 @@
       - CableStack
       - HandheldGPSBasic
       - HandheldHealthAnalyzer
+      - AirlockPainter
+  - type: Appearance
+    visuals:
+    - type: AutolatheVisualizer
+    - type: WiresVisualizer
```

Comment:
Merge artifact?

Reply:
I just realized the issue, sorry about that I got confused earlier. This should hopefully fix it